communitiesuk/funding-service-design-fund-application-builder
# FS-4916 - Add Sentry monitoring to FAB

### Ticket

[Add Sentry monitoring to FAB](https://mhclgdigital.atlassian.net/browse/FS-4916)

### Description

This change adds Sentry monitoring to FAB, using `init_sentry` from the fsd-utils library. A [new project called fund-application-builder](https://funding-service-design-team-dl.sentry.io/projects/fund-application-builder/?project=4508496706666497) has been set up in the Sentry web UI, and this is what is referenced by `SENTRY_DSN`. The uv.lock changes were automatically applied with `uv lock` after the pyproject.toml single-line change.

### How to test

- Deploy to dev environment
- Go to https://fund-application-builder.dev.access-funding.test.levellingup.gov.uk/dev/test_sentry
- Go to #funding-service-non-prod-alerts Slack channel
- You should see an error message
